src/time/daycounters/thirty360.rs

```diff
@@ -146,19 +151,27 @@ impl Thirty360 {
     }
 }
 
+impl Default for Thirty360 {
+    fn default() -> Self {
+        Self {
+            market: Thirty360Market::USA,
+        }
+    }
+}
+
 #[typetag::serialize]
 impl DayCounters for Thirty360 {
     fn day_count(&self, d1: NaiveDate, d2: NaiveDate) -> i64 {
```

Comment:
@thejerwang I'm of two minds whether this should return a signed integer? I feel like `Result<u64>` may be more appropriate (with `end_date < start_date` returning `Err`), although I see that QuantLib also returns signed integers. Thoughts on this?